lib/rewards-bamboohr/templates/anniversary.rb

```diff
@@ -0,0 +1,45 @@
+module RewardsBamboohr
+  module Templates
+    class Anniversary < Base
+      DEFAULT_TEMPLATE = "+100 Happy %{number} Anniversary @%{username}".freeze
+
+      def generate
+        template % { username: username, number: anniversary_number }
+      end
+
+      private
+
+      def anniversary_number
+        ordinalize(year(Date.today) - year(bamboohr_data["hireDate"]))
+      end
+
+      def year(date)
+        return unless date
+
+        date = Date.parse(date) unless date.is_a? Date
+        date.strftime("%Y").to_i
+      end
+
+      def ordinal(number)
```

Comment:
Do you think we could use https://devdocs.io/rails~5.1/activesupport/inflector#method-i-ordinal?

Reply:
We could, but I've tried to avoid adding ActiveSupport here.

Reply:
ActiveSupoprt already in the deps

Reply:
Ok